mlrun/runtimes/pod.py

```diff
@@ -330,6 +333,10 @@ def _verify_and_set_limits(
                         resource_value,
                     )
 
+    def rollback_fields(self):
+        for k, v in self.__fields_pending_rollback.items():
+            setattr(self, k, v)
```

Comment:
maybe discard_changes?
we need to doc that function as it is public. perhaps explain the intention and why it is needed (I guess can be taken from PR)

Reply:
Indeed. I added added a docstring for us. Please take a look to see if it's appropriate.